### PR TITLE
Prevent the code from trying to add users to board cameos multiple times

### PR DESCRIPTION
**Pull request by teceler**
(Also includes a test demonstrating the problem)

### CODE REVIEW

File: app/concerns/owable.rb [hunk 1]
```diff
@@ -48,7 +48,7 @@ def update_board_cameos
 
       # adjust for the fact that the associations are managed separately
       all_authors = authors + unjoined_authors + joined_authors + tagging_authors
-      new_cameos = all_authors.uniq - board.writers
+      new_cameos = all_authors.uniq - (board.writers + board.cameos)
```
Comment (Marri) [inline app/concerns/owable.rb]: Does board.authors include board.creator? Because if it does we can use that and then we don't need to do the addition.

Comment (teceler) [reply app/concerns/owable.rb]: board.authors seems to not exist, actually! it's just board.coauthors and board.cameos

Comment (Marri) [reply app/concerns/owable.rb]: sorry, I meant board.board_authors

Comment (teceler) [reply app/concerns/owable.rb]: Ah. Having poked at Effulgence in my dev env, the creator doesn't seem to have a BoardAuthor object.

Comment (Marri) [reply app/concerns/owable.rb]: Blah okay, we'll fix it if we ever get that sorted

**Pull request merged**